sympy/physics/control/lti.py

```diff
@@ -378,27 +400,98 @@ def is_biproper(self):
 
 
 class Series(Basic):
+    """
+    A class for representing product of transfer functions (or ``Parallel`` objects).
+    All the transfer functions (or ``Parallel`` objects) should be using the same complex variable ``var`` of the
+    Laplace transform.
+
+    Examples
+    ========
+
+    >>> from sympy.abc import s, p, a, b
+    >>> from sympy.physics.control.lti import TransferFunction, Series, Parallel
+    >>> tf1 = TransferFunction(a*p**2 + b*s, s - p, s)
+    >>> tf2 = TransferFunction(s**3 - 2, s**4 + 5*s + 6, s)
+    >>> tf3 = TransferFunction(p**2, p + s, s)
+    >>> S1 = Series(tf1, tf2)
+    >>> S1
+    Series(TransferFunction(a*p**2 + b*s, -p + s, s), TransferFunction(s**3 - 2, s**4 + 5*s + 6, s))
+    >>> S1.var
+    s
+    >>> S2 = Series(tf2, Parallel(tf3, -tf1))
+    >>> S2
+    Series(TransferFunction(s**3 - 2, s**4 + 5*s + 6, s), Parallel(TransferFunction(p**2, p + s, s), TransferFunction(-a*p**2 - b*s, -p + s, s)))
+    >>> S2.var
+    s
+
+    You can get the resultant transfer function by using ``.doit()`` method:
 
+    >>> S3 = Series(tf1, tf2, -tf3)
+    >>> S3.doit()
+    TransferFunction(-p**2*(s**3 - 2)*(a*p**2 + b*s), (-p + s)*(p + s)*(s**4 + 5*s + 6), s)
+    >>> S4 = Series(tf2, Parallel(tf1, -tf3))
+    >>> S4.doit()
+    TransferFunction((s**3 - 2)*(-p**2*(-p + s) + (p + s)*(a*p**2 + b*s)), (-p + s)*(p + s)*(s**4 + 5*s + 6), s)
+
+    See Also
+    ========
+
+    sympy.physics.control.lti.Parallel
+
+    """
     def __new__(cls, *args, evaluate=False):
         if not all(isinstance(arg, (TransferFunction, Parallel, Series)) for arg in args):
-            raise TypeError("Unsupported type of argument(s) for Series.")
+            raise ValueError("Unsupported type of argument(s) for Series.")
 
         obj = super(Series, cls).__new__(cls, *args)
         obj._var = None
         for arg in args:
             if obj._var is None:
                 obj._var = arg.var
             elif obj._var != arg.var:
-                raise ValueError("All transfer functions should be anchored with the same variable.")
+                raise ValueError("All transfer functions should use the same complex variable of the Laplace transform.")
         if evaluate:
             return obj.doit()
         return obj
 
     @property
     def var(self):
+        """
+        Returns the complex variable used by all the transfer functions (or ``Parallel`` objects).
```

Comment:
```suggestion
        Returns the complex variable used by all the transfer functions (or ``Series`` objects).
```

Reply:
This is a property of `Series` class, it's obvious that it will return the complex variable used by all the "Series" objects. What I meant was this:
``` python
>>> Series(tf1, tf2, Parallel(tf3, tf4)).var
s
```
I will just change it like that,  if it isn't explicit.
```suggestion
        Returns the complex variable used by all the transfer functions.
```